src/net_processing.cpp

```diff
@@ -195,6 +197,11 @@ static constexpr double MAX_ADDR_RATE_PER_SECOND{0.1};
 static constexpr size_t MAX_ADDR_PROCESSING_TOKEN_BUCKET{MAX_ADDR_TO_SEND};
 /** The compactblocks version we support. See BIP 152. */
 static constexpr uint64_t CMPCTBLOCKS_VERSION{2};
+/** How frequently to update templates for sharing */
+static constexpr std::chrono::microseconds TEMPLATE_UPDATE_INTERVAL{30s};
+/** Template weight limit */
+static constexpr unsigned int MAX_TEMPLATE_WEIGHT{8000000};
```

Comment:
Does `MAX_PROTOCOL_MESSAGE_LENGTH` need to change if a peer requests most txns in a > 4MB template?